### PR TITLE
Dnf

### PR DESCRIPTION
**Pull request by tobes**
Somehow I stupidly deleted my dnf branch rather than the batter_power one

Anyhow I think this is all back as it should be.  Sorry about that

### CODE REVIEW

File: py3status/modules/fedora_updates.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Displays the number of package updates pending for a Fedora Linux installation.
+This will display a count of how many `dnf` updates are waiting
```
Comment (ultrabug) [inline py3status/modules/fedora_updates.py]: please add a blank line above this line (separates the short description with the long one)

**Comment by ultrabug** [comment]
Also, please make sure your code passes **flake8** so the travis-ci above passes

**Comment by tobes** [comment]
done

**Comment by ultrabug** [comment]
Thanks for your contribution @tobes !

**Pull request merged**